dandischema/tests/test_datacite.py

```diff
@@ -0,0 +1,277 @@
+from datetime import datetime
+import json
+import os
+from pathlib import Path
+import random
+
+from jsonschema import Draft6Validator
+import pytest
+import requests
+
+from ..datacite import to_datacite
+from ..models import LicenseType, PublishedDandisetMeta, RoleType
+
+
+def datacite_post(datacite, doi):
+    """ posting the datacite object and checking the status of the requests"""
```

Comment:
```suggestion
    """Post the datacite object and check the status of the request"""
```
I think imperative form is generally better and that is what we use primarily: it is shorter, `-ing` suggests some extended in time action (content manager? generator? nope...), and I do not see a point in leading space.